src/commands/env/compute/collaborator/add.ts

```diff
@@ -74,5 +72,8 @@ export default class ComputeCollaboratorAdd extends Command {
     this.log(
       'For more information about attaching Heroku add-ons to your compute environments, run $ heroku addons:attach --help.'
     );
+    return {
```

Comment:
If the user includes the --json flag, the error output should be in json also.

Try adding in:
```
if (shouldExitNonZero) {
      this.exit(1);
    }
```

Reply:
Example here https://github.com/salesforcecli/plugin-functions/blob/e63ff43943c0943609126acb33d21e66d6d35818/src/commands/deploy/functions.ts#L188